src/routers/collections.py

```
@@ -6,16 +6,41 @@
 )
 
 
-@router.post("/list", response_model=Page[schemas.Collection])
+@router.get(
+    "",
+    response_model=schemas.Collection,
+)
+async def get_collection_from_token(
+    app_id: str,
+    user_id: str,
+    jwt_params: JWTParams = Depends(auth),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)